manifests/repo.pp

```diff
@@ -22,4 +22,14 @@
     enabled  => true,
   }
   -> anchor { 'katello::repo': } # lint:ignore:anchor_resource
+
+  if $facts['os']['release']['major'] == '8' and ($repo_version == 'nightly' or versioncmp($repo_version, '4.3') >= 0) {
+    package { 'katello-dnf-module':
+      ensure      => "el${facts['os']['release']['major']}",
```

Comment:
Should this be
```suggestion
      ensure      => $dist,
```

Reply:
probably, even tho I don't really understand this param… like, why would you override it?

Reply:
I think it mostly stems from Amazon Linux 2 which reports a major version of 2 but really is el7. This is sort of a compromise: you can override it but we don't provide a decent mapping. There's also no harm in defining it this was rather than as a static variable inside the body.